guides/type_definitions/lists.md

```diff
@@ -109,7 +109,7 @@ Valid | Invalid
 ------|------
 `[]`  | `null`
 `[1, 2, ...]`| `[null]`
- | `[1, null, 2, ...]`
+| | `[1, null, 2, ...]`
```

Comment:
Note the original intention here - you can see that this is meant to be in the second column (invalid)

Adding a second `|` helps markdown realize that this is meant to be in column two.